rgbd/src/client.cpp

```diff
@@ -0,0 +1,58 @@
+#include "rgbd/client.h"
+
+#include <opencv2/core/mat.hpp>
+
+#include "rgbd/image.h"
+#include "rgbd/ros/conversions.h"
+
+namespace rgbd {
+
+// ----------------------------------------------------------------------------------------
```

Comment:
If you're writing new files anyway: I don't think these dividers are very common in cpp files

Reply:
I could remove these in the entire repo